models/enrollment_journey_model.sql

```diff
@@ -0,0 +1,42 @@
+{{ config(
+  materialized='table'
+) }}
+
+WITH enrollment_data AS (
+    SELECT
+        er.academic_year,
+        er.is_current,
+        er.start_date AS start_date,
+        er.end_date AS end_date,
+        er.inserted_at AS record_inserted_at,
+        er.updated_at AS record_updated_at,
+        er.user_id,
+        er.subject_id,
+        er.grade_id,
+        er.group_type,
+        er.group_id
+    FROM
+        {{ source('source_postgres', 'enrollment_record') }} AS er
+    WHERE
+        er.is_current = true
+)
```

Comment:
can't we have this stored as a separate model? something like `current_enrollment_data` and then reference it in the query below? Because we might need this a lot of times no?

Reply:
The idea behind this table was to track student movement across batch, school and auth_group